stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -17,6 +19,26 @@ Follow the steps below to ensure a smooth backup and recovery process.
 It is important to take a manual snapshot over relying on the automated ones created by an instance as automated snapshots are attached to the lifecycle of the RDS instance. 
 You cannot use an automated snapshot for recovery in the event that your deployment is lost.
 
+Create an On-Demand Backup For the EFS of Your Existing Deployment
```

Comment:
Change to:
== Creating an on -demand backup for the EFS of your existing deployment

To make it a sub-heading like the ones following.